README.yaml

```diff
@@ -78,18 +78,39 @@ usage: |2-
       source = "cloudposse/elasticache-memcached/aws"
       # Cloud Posse recommends pinning every module to a specific version
       # version = "x.x.x"
-      namespace               = var.namespace
-      stage                   = var.stage
-      name                    = var.name
-      availability_zones      = var.availability_zones
-      vpc_id                  = module.vpc.vpc_id
-      allowed_security_groups = [module.vpc.vpc_default_security_group_id]
-      subnets                 = module.subnets.private_subnet_ids
-      cluster_size            = var.cluster_size
-      instance_type           = var.instance_type
-      engine_version          = var.engine_version
-      apply_immediately       = true
-      zone_id                 = var.zone_id
+
+      namespace          = var.namespace
+      stage              = var.stage
+      name               = var.name
+      availability_zones = var.availability_zones
+      vpc_id             = module.vpc.vpc_id
+      subnets            = module.subnets.private_subnet_ids
+      cluster_size       = var.cluster_size
+      instance_type      = var.instance_type
+      engine_version     = var.engine_version
+      apply_immediately  = true
+      zone_id            = var.zone_id
+
+      security_group_rules = [
+        {
+          type                     = "egress"
+          from_port                = 0
+          to_port                  = 65535
+          protocol                 = "-1"
+          cidr_blocks              = ["0.0.0.0/0"]
+          source_security_group_id = null
+          description              = "Allow all outbound traffic"
+        },
+        {
+          type                     = "ingress"
+          from_port                = 0
+          to_port                  = 65535
```

Comment:
This example is pretty open security-wise. It's pretty clear to the user so I'm sure they would be more aware of it but it would be nice if the example itself provided ingress to only the port it required. For instance, `var.port` is an input variable here, we could easily set the `var.port` to a local and provide it to `var.port` and the `from_port` and `to_port`.